internal/store/namespace.go

```diff
@@ -32,6 +32,10 @@ var (
 	descNamespaceLabelsHelp          = "Kubernetes labels converted to Prometheus labels."
 	descNamespaceLabelsDefaultLabels = []string{"namespace"}
 
+	descNamespaceAnnotationsName          = "kube_namespace_annotations"
```

Comment:
Seems like we already had the `kube_namespace_annotations` metric before this addition, but this metric is marked as STABLE in our docs. Any suggestions for this? We can either remove it, but because it's marked as stable, I would say we try to filter out?

Reply:
We discussed this and we want to remove this.